lighthouse-cli/test/smokehouse/smokehouse-report.js

```diff
@@ -194,7 +199,8 @@ function reportAssertion(assertion) {
   } else {
     if (assertion.diff) {
       const diff = assertion.diff;
-      const fullActual = JSON.stringify(assertion.actual, null, 2).replace(/\n/g, '\n      ');
+      const fullActual = String(JSON.stringify(assertion.actual, null, 2))
```

Comment:
fixes the case when `assertion.actual` is `undefined`, which surprisingly I guess we've never encountered before because this throws on `undefined.replace(...)`